routers/web/admin/applications.go

```diff
@@ -23,8 +23,8 @@ var (
 func newOAuth2CommonHandlers() *user_setting.OAuth2CommonHandlers {
 	return &user_setting.OAuth2CommonHandlers{
 		OwnerID:            0,
-		BasePathList:       fmt.Sprintf("%s/admin/applications", setting.AppSubURL),
-		BasePathEditPrefix: fmt.Sprintf("%s/admin/applications/oauth2", setting.AppSubURL),
+		BasePathList:       fmt.Sprintf("%s%s/applications", setting.AppSubURL, setting.AdminRouterPrefix),
+		BasePathEditPrefix: fmt.Sprintf("%s%s/applications/oauth2", setting.AppSubURL, setting.AdminRouterPrefix),
```

Comment:
The const bring no benefit but only make the code more complex.

Even if you introduce a const, there are still `/-/admin` strings in templates.

----

* With a const, it needs to search both `AdminRouterPrefix` and `/-/admin`.
* Without a const, it only needs to search  `/-/admin` which is simpler.

Reply:
ea92b94a0e6873e24747360c376d25e2d606069e